client/my-sites/upgrades/domain-management/name-servers/dns-template-button.jsx

```diff
@@ -1,14 +1,14 @@
 /**
  * External dependencies
  */
-import React, { Component } from 'react';
+import React from 'react';
 
 /**
  * Internal dependencies
  */
 import Button from 'components/button';
 
-class DnsTemplateButton extends Component {
+class DnsTemplateButton extends Button {
```

Comment:
I'm not sure if this is correct or not.  I did this because ButtonGroup logs a warning if it contains instances of any component except Buttons.